src/pvecontrol/sanitycheck/tests/vm_backups.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime, timedelta
+from pvecontrol.sanitycheck.checks import Check, CheckCode, CheckType, CheckMessage
+
+
+class VmBackups(Check):
+
+    id = "vm_backups"
+    type = CheckType.VM
+    name = "Check that Vms are backup up on a regular basis"
+
+    def run(self):
+        backuped_vms = self._check_is_backed_up()
+        self._check_backup_ran_recently(backuped_vms)
+
+    def _check_is_backed_up(self):
+        backuped_vms = []
+        for vm in self.proxmox.vms:
+            vm_backup_jobs = vm.get_backup_jobs(self.proxmox)
+            vm_enabled_backup_ids = [backup.id for backup in vm_backup_jobs if backup.enabled == 1]
+            if len(vm_enabled_backup_ids) > 0:
+                msg = f"Vm {vm.vmid} ({vm.name}) is associated to {len(vm_enabled_backup_ids)} enabled backup job(s)"
+                self.add_messages(CheckMessage(CheckCode.OK, msg))
+                backuped_vms.append(vm)
+            else:
+                msg = f"Vm {vm.vmid} ({vm.name}) is not associated to any backup job"
+                self.add_messages(CheckMessage(CheckCode.CRIT, msg))
```

Comment:
I will set Warning here. As a VM without a backup is not a broken status, this can be legit.